lib/both.js

```diff
@@ -73,10 +73,8 @@ DocHead = {
   removeDocHeadAddedTags() {
     if (Meteor.isClient) {
       const elements = document.querySelectorAll('[dochead="1"]');
-      for (let key in elements) {
-        if (elements.hasOwnProperty(key)) {
-          elements[key].parentNode.removeChild(elements[key]);
-        }
+      for (let element of elements) {
+        element.remove();
```

Comment:
It seems like element.remove() is a new API only works on new browsers.
See: https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove

We need to find something works upto IE9